backend/src/api/users/users.api.ts

```diff
@@ -21,6 +23,22 @@ const initUsersApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
       return rep.status(HttpCode.OK).send(users);
     },
   });
+
+  fastify.route({
+    method: HttpMethod.DELETE,
+    url: UsersApiPath.$ID,
+    schema: { params: userDeleteRequestParamsValidationSchema },
+    async handler(
+      req: FastifyRequest<{ Params: UserDeleteRequestParamsDto }>,
+      rep,
+    ) {
+      const { id } = req.params;
+
+      const isDeleted = await userService.delete(Number(id));
+
+      return rep.status(isDeleted ? HttpCode.OK : HttpCode.NOT_FOUND);
```

Comment:
delete should return 204 code if deleted entity isn't supplied
[https://www.rfc-editor.org/rfc/rfc9110.html#DELETE](https://www.rfc-editor.org/rfc/rfc9110.html#DELETE)